src/OpenSage.Game/Logic/Object/Draw/W3dFloorDraw.cs

```diff
@@ -104,6 +104,8 @@ public sealed class W3dFloorDrawModuleData : DrawModuleData
 
         internal override DrawModule CreateDrawModule(GameObject gameObject, GameContext context)
         {
+            if (Model.Value == null) return null;
```

Comment:
a model "mirkBridge" references a model 'NBMirkBridge_Bib' which is not present in the game assets.

Reply:
We should atleast log a warning here

Reply:
Since the model is null here we can not really log a model name. Thus the log message would only be e.g. "a non existing model was referenced in a W3DFloorDraw module" -> Does not really help.